Resources/Prototypes/Catalog/Fills/Lockers/security.yml

```
@@ -3,243 +3,349 @@
   suffix: Filled, Hardsuit
   parent: LockerWarden
   components:
-  - type: StorageFill
-    contents:
-      - id: FlashlightSeclite
-      - id: WeaponDisabler
-      - id: ClothingBeltSecurityFilled
-      - id: Flash
-      - id: ClothingEyesGlassesSecurity
-      - id: ClothingHandsGlovesCombat
-      - id: ClothingShoesBootsJack
-      - id: ClothingOuterCoatWarden
-      - id: ClothingOuterWinterWarden
-      - id: RubberStampWarden
-      - id: DoorRemoteArmory
-      - id: ClothingOuterHardsuitWarden
-      - id: HoloprojectorSecurity
-      - id: BookSpaceLaw
-      - id: ClothingBackpackElectropack
-        amount: 2
-      - id: RemoteSignaller
-        amount: 2
+  - type: EntityTableContainerFill
+    containers:
+      entity_storage: !type:AllSelector
+        children:
+        - !type:NestedSelector
+          tableId: FillLockerWarden
+        - !type:NestedSelector
+          tableId: FillLockerWardenHarduit
 
 - type: entity
   id: LockerWardenFilled
   suffix: Filled
   parent: LockerWarden
   components:
-  - type: StorageFill
-    contents:
-      - id: FlashlightSeclite
-      - id: WeaponDisabler
-      - id: ClothingBeltSecurityFilled
-      - id: Flash
-      - id: ClothingEyesGlassesSecurity
-      - id: ClothingHandsGlovesCombat
-      - id: ClothingShoesBootsJack
-      - id: ClothingOuterCoatWarden
-      - id: ClothingOuterWinterWarden
-      - id: RubberStampWarden
-      - id: DoorRemoteArmory
-      - id: HoloprojectorSecurity
-      - id: BookSpaceLaw
-      - id: ClothingBackpackElectropack
-        amount: 2
-      - id: RemoteSignaller
-        amount: 2
-      - id: Binoculars
+  - type: EntityTableContainerFill
+    containers:
+      entity_storage: !type:NestedSelector
+        tableId: FillLockerWarden
+
+- type: entityTable
+  id: FillLockerWarden
+  table: !type:AllSelector
+    children:
+    - id: FlashlightSeclite
+    - id: WeaponDisabler
+    - id: ClothingBeltSecurityFilled
+    - id: Flash
+    - id: ClothingEyesGlassesSecurity
+    - id: ClothingHandsGlovesCombat
+    - id: ClothingShoesBootsJack
+    - id: ClothingOuterCoatWarden
+    - id: ClothingOuterWinterWarden
+    - id: RubberStampWarden
+    - id: DoorRemoteArmory
+    - id: HoloprojectorSecurity
+    - id: BookSpaceLaw
+    - id: ClothingBackpackElectropack
+      amount: !type:ConstantNumberSelector
+        value: 2
+    - id: RemoteSignaller
+      amount: !type:ConstantNumberSelector
+        value: 2
```

You don't need to use these constant number selectors, the serializer assumes you're using a constant number.

```suggestion
    - id: ClothingBackpackElectropack
      amount: 2
    - id: RemoteSignaller
      amount: 2
```